### PR TITLE
adds print statements so the NutritionDisplay has a temp view

### LINKED ISSUE #27: Make temp view (print statements) for NutritionDisplay
as title says.

### PR DESCRIPTION
**Pull request by parkhangorodsky**
Basically, right now the SearchRecipe use case gets called when a user searches something, and then for each recipe in the result, the NutritionDisplay use case is called and should update the SearchRecipeView. But that makes two separate calls to change the View for what should realistically happen together. We need some major structural changes - but this wasn't possible before since the User class wasn't done so it wasn't clear what the use cases for the NutritionAPI would entail. So, rather than making a temporary View process for now and then changing everything after Phase 1, I added some print statements to show that the NutritionDisplay use case works.

These changes will be removed later once we fix the flow of information.

### CODE REVIEW

**Review by RqedK — APPROVED** [review]
Looks good, no conflict with my usecases/branch

File: src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java [hunk 1]
```diff
@@ -10,13 +10,19 @@
...
+         // temp line for the temp view:
+         System.out.println("The recipe has the following nutritional info:");
+
          List<Nutrition> nutritions = new ArrayList<>();
 
          if (nutritionJSONObject.has("error")) {
              nutritions.add(new Nutrition());
          } else {
              float calories = nutritionJSONObject.getFloat("calories");
-             nutritions.add(new Nutrition("calories", calories, "kcal"));
```
Comment (hhpfiona) [inline src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java:19]: easier to read, nice

**Review by hhpfiona — APPROVED** [review]

File: src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java [hunk 2]
```diff
@@ -34,10 +40,18 @@ default List<Nutrition> convertJSONtoNutritionList(JSONObject nutritionJSONObjec
...
-                 nutritions.add(new Nutrition(label, quantity, unit));
+                 Nutrition nutritionFact = new Nutrition(label, quantity, unit);
+                 nutritions.add(nutritionFact);
+
+                 // temp line for the temp view:
+                 System.out.println(nutritionFact);
              }
          }
-
+         // temp lines for the temp view:
```
Comment (seonghyunban) [inline src/main/java/use_cases/_common/xtra/json_processor/NutritionJSONHandler.java:50]: Looks good

**Review by seonghyunban — APPROVED** [review]
Looks good!

**Pull request merged**